nextflow.config

```diff
@@ -0,0 +1,87 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    UMCUGenetics/GIABEval Nextflow config file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+manifest {
+    name            = 'UMCUGenetics/GIABEval'
+    author          = """UMCU Genetics"""
+    homePage        = 'https://github.com/UMCUGenetics/GIABEval'
+    description     = """UMCU Genetics Genome-in-a-bottle evaluation workflow for benchmarking"""
+    mainScript      = 'main.nf'
+    nextflowVersion = '!>=23.04.0'
```

Comment:
Right now, this version requirement conflicts with the version of the Nextflow installation that is used in `run_nextflow_giabeval.sh`. It might be better to use an installation of Nextflow specific for this workflow.

Reply:
Agree